app/event/send-acknowledgement-event.js

```diff
@@ -0,0 +1,12 @@
+const sendInvalidBankDetailsEvent = require('./send-invalid-bank-details-event')
+const sendProcessingAckErrorEvent = require('./send-processing-ack-error-event')
+
+const sendAcknowledgementEvent = async (holdCategoryName, acknowledgement, frn) => {
```

Comment:
Also, would naming this `sendAcknowledgementErrorEvent` make it clearer that these events are sent based on ack errors not successes?

Reply:
I assume that would include renaming the file itself on top of renaming this function?

Reply:
Yup